module/entityAssessors/helper.js

```diff
@@ -250,12 +265,11 @@ module.exports = class entityAssessorHelper {
                     let entityAssessorDocument = {
                         "action": assessor.entityOperation,
                         "entities": assessor.entities,
-                        "type": "ASSESSOR",
                         "assessorId": assessor.userId,
                         "programId": assessor.programId
                     }
 
-                    await this.uploadAssessorSchoolTracker(entityAssessorDocument)
+                    await this.uploadEntityAssessorTracker(entityAssessorDocument)
```

Comment:
Add on line 270 condition fo solution and program id.

@ajithtunerlabs

Reply:
added programId and solutionId as query object

Reply:
addressed